routes/coverage.js

```diff
@@ -89,11 +94,20 @@ var saveCoverage = function ( hash, coverage, coverageType, options, callback )
                 return compare.hash === hash;
             } );
 
-            priorHash = indexOfCurrent === -1
-                ? hashes[ 0 ]
-                : hashes.length > indexOfCurrent + 1
-                    ? hashes[ indexOfCurrent + 1 ]
-                    : null;
+            var getPriorHashOnMaster = function ( indexOfCurrent )
```

Comment:
If I'm reading this correctly, it goes back through the cvr-held history to the most recent non-PR coverage.  Assuming that's correct:

- that non-PR coverage may or may not be on `master` specifically—this is really just an issue of the name of the function.
- that coverage could be on an entirely different branch and not an ancestor of the current PR—I don't really know how to fix this, though.